FilmFestivalLoader/MTMF/parse_mtmf_html.py

```diff
@@ -503,17 +551,43 @@ def set_screening_times(self, data):
 
     def set_screen(self, data):
         items = data.split(',')    # Den Haag, Filmhuis Den Haag
-        city = items[0] or home_city
+        city = items[0] or FESTIVAL_CITY
         theater = items[1].strip()
-        screen_name = self.screen_name if self.screen_name is not None else theater
-        if screen_name is not None:
-            self.screen = self.festival_data.get_screen(city, screen_name, theater)
+        if self.sold_out:
+            counter.increase('sold out')
+            self.screen = self.get_screen_from_file()
+            if not self.screen:
+                error_collector.add('Screen not recovered', f'{city=}, {theater=}, {self.start_dt.isoformat(sep=" ")}')
+                counter.increase('screen not recovered')
         else:
-            self.print_debug('NO THEATER', f'city={city}, theater={theater}, screen={screen_name}')
-            counter.increase('no theater')
-        if city != home_city:
+            screen_name = self.screen_name if self.screen_name is not None else theater
+            if screen_name is not None:
+                self.screen = self.festival_data.get_screen(city, screen_name, theater)
+            else:
+                self.print_debug('NO THEATER', f'city={city}, theater={theater}, screen={screen_name}')
+                counter.increase('no theater')
+        if city != FESTIVAL_CITY:
             self.print_debug('OTHER CITY', f'city={city}, theater={theater}, screen={self.screen}')
 
+    def get_screen_from_file(self):
+        film_id_field = 0
+        screen_id_field = 1
+        start_time_field = 2
```

Comment:
PyCharm wouldn't let me do it: "variable in function should be lowercase".

Alternatives like module constants or even class constants would take the definitions too far from their use. This would more or less force us to expand the names with something like the model name.

A bit of an overkill, so won't do.